.gitlab-ci.yml

```diff
@@ -77,7 +77,7 @@ execute:
     -   make destroy
     - else
     -   git checkout $STAKATER_PLATFORM_BRANCH
-    -   make setupvars CLOUD_PROVIDER=$CLOUD_PROVIDER STAKATER_PLATFORM_SSH_GIT_URL=$STAKATER_PLATFORM_SSH_GIT_URL STAKATER_PLATFORM_BRANCH=$STAKATER_PLATFORM_BRANCH DOMAIN=$DOMAIN BASE64_ENCODED_AWS_ACCESS_KEY_ID=$BASE64_ENCODED_AWS_ACCESS_KEY_ID BASE64_ENCODED_AWS_SECRET_ACCESS_KEY=$BASE64_ENCODED_AWS_SECRET_ACCESS_KEY BASE64_ENCODED_IMC_CONFIG=$BASE64_ENCODED_IMC_CONFIG BASE64_ENCODED_JENKINS_CFG=$BASE64_ENCODED_JENKINS_CFG KEYCLOAK_CLIENT_ID=$KEYCLOAK_CLIENT_ID KEYCLOAK_CLIENT_SECRET=$KEYCLOAK_CLIENT_SECRET BASE64_ENCODED_HUB_TOKEN=$BASE64_ENCODED_HUB_TOKEN BASE64_ENCODED_GITLAB_TOKEN=$BASE64_ENCODED_GITLAB_TOKEN BASE64_ENCODED_BITBUCKET_TOKEN=$BASE64_ENCODED_BITBUCKET_TOKEN BASE64_ENCODED_ADMIN_ACCOUNT_JSON=$BASE64_ENCODED_ADMIN_ACCOUNT_JSON BASE64_ENCODED_CLUSTER_ACCOUNT_JSON=$BASE64_ENCODED_CLUSTER_ACCOUNT_JSON NEXUS_ADMIN_ACCOUNT_USERNAME=$NEXUS_ADMIN_ACCOUNT_USERNAME NEXUS_CLUSTER_ACCOUNT_USERNAME=$NEXUS_CLUSTER_ACCOUNT_USERNAME BASE64_ENCODED_ALERTMANAGER_CONFIG=$BASE64_ENCODED_ALERTMANAGER_CONFIG BASE64_ENCODED_KEYCLOAK_CONFIG=$BASE64_ENCODED_KEYCLOAK_CONFIG BASE64_ENCODED_PROXYINJECTOR_CONFIG=$BASE64_ENCODED_PROXYINJECTOR_CONFIG BASE64_ENCODED_SLACK_CHANNEL=$BASE64_ENCODED_SLACK_CHANNEL BASE64_ENCODED_SLACK_WEBHOOK_URL=$BASE64_ENCODED_SLACK_WEBHOOK_URL
+    -   make configure CLOUD_PROVIDER=$CLOUD_PROVIDER STAKATER_PLATFORM_SSH_GIT_URL=$STAKATER_PLATFORM_SSH_GIT_URL STAKATER_PLATFORM_BRANCH=$STAKATER_PLATFORM_BRANCH DOMAIN=$DOMAIN BASE64_ENCODED_AWS_ACCESS_KEY_ID=$BASE64_ENCODED_AWS_ACCESS_KEY_ID BASE64_ENCODED_AWS_SECRET_ACCESS_KEY=$BASE64_ENCODED_AWS_SECRET_ACCESS_KEY BASE64_ENCODED_IMC_CONFIG=$BASE64_ENCODED_IMC_CONFIG BASE64_ENCODED_JENKINS_CFG=$BASE64_ENCODED_JENKINS_CFG KEYCLOAK_CLIENT_ID=$KEYCLOAK_CLIENT_ID KEYCLOAK_CLIENT_SECRET=$KEYCLOAK_CLIENT_SECRET BASE64_ENCODED_GITHUB_TOKEN=$BASE64_ENCODED_GITHUB_TOKEN BASE64_ENCODED_GITLAB_TOKEN=$BASE64_ENCODED_GITLAB_TOKEN BASE64_ENCODED_BITBUCKET_TOKEN=$BASE64_ENCODED_BITBUCKET_TOKEN BASE64_ENCODED_ADMIN_ACCOUNT_JSON=$BASE64_ENCODED_ADMIN_ACCOUNT_JSON BASE64_ENCODED_CLUSTER_ACCOUNT_JSON=$BASE64_ENCODED_CLUSTER_ACCOUNT_JSON NEXUS_ADMIN_ACCOUNT_USERNAME=$NEXUS_ADMIN_ACCOUNT_USERNAME NEXUS_CLUSTER_ACCOUNT_USERNAME=$NEXUS_CLUSTER_ACCOUNT_USERNAME BASE64_ENCODED_ALERTMANAGER_CONFIG=$BASE64_ENCODED_ALERTMANAGER_CONFIG BASE64_ENCODED_KEYCLOAK_CONFIG=$BASE64_ENCODED_KEYCLOAK_CONFIG BASE64_ENCODED_PROXYINJECTOR_CONFIG=$BASE64_ENCODED_PROXYINJECTOR_CONFIG BASE64_ENCODED_SLACK_CHANNEL=$BASE64_ENCODED_SLACK_CHANNEL BASE64_ENCODED_SLACK_WEBHOOK_URL=$BASE64_ENCODED_SLACK_WEBHOOK_URL
```

Comment:
i think  these  `BASE64_ENCODED_JENKINS_CFG` should be converted to `BASE64_ENCODED_JENKINS_CONFIG`. I think we should follow these conventions everywhere

Reply:
Ignore `makefile-ci` for now. We'll merge all of this into `makefile` eventually to keep things consistent